models/GitHub.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: connor
+ * Date: 15/07/2016
+ * Time: 13:36
+ */
+
+namespace QA;
+
+class GitHub
+{
+    private $accessToken;
+
+    const CLIENT_ID = '3abdc7f847c577e30725';
```

Comment:
This data can be stored in a config file which is loaded with PHP. It'll make easier to update the data without touching PHP code. It'll also make ID e SECRET reusable in case other classes need them.